.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
```

Comment:
Just a note for later -- probably should rename this. I'm guessing this template was originally for a backend Node.js app.